audit/src/main/scala/tech/beshu/ror/audit/enrichers/AuditLogSerializerEnrichedWithEsNodeDetails.scala

```diff
@@ -0,0 +1,31 @@
+/*
+ *    This file is part of ReadonlyREST.
+ *
+ *    ReadonlyREST is free software: you can redistribute it and/or modify
+ *    it under the terms of the GNU General Public License as published by
+ *    the Free Software Foundation, either version 3 of the License, or
+ *    (at your option) any later version.
+ *
+ *    ReadonlyREST is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *    GNU General Public License for more details.
+ *
+ *    You should have received a copy of the GNU General Public License
+ *    along with ReadonlyREST.  If not, see http://www.gnu.org/licenses/
+ */
+package tech.beshu.ror.audit.enrichers
+
+import tech.beshu.ror.audit.AuditLogSerializer
+
+class AuditLogSerializerEnrichedWithEsNodeDetails(underlying: AuditLogSerializer,
```

Comment:
Currently, we configure serializers like that:
```
readonlyrest:
  audit:
    enabled: true
    outputs:
    - type: index
      serializer: tech.beshu.ror.requestcontext.QueryAuditLogSerializer
```

So it'd be nice if users could do the following:
```
serializer tech.beshu.ror.audit.enrichers.AuditLogSerializerEnrichedWithEsNodeDetails
```

Reply:
Answered in other dicsussion - introduced versioning, and the `DefaultAuditLogSerializer` is an alias for the newest version